baselines/dlrm_main.py

```diff
@@ -555,9 +567,46 @@ def main(argv: List[str]) -> None:
         "learning_rate": args.learning_rate,
         "optimizer": OptimType.EXACT_ROWWISE_ADAGRAD if args.adagrad else OptimType.EXACT_SGD,
     }
-    sharders = [
-        EmbeddingBagCollectionSharder(fused_params=fused_params),
-    ]
+
+    if args.sharder_type == 'naive':    
+        class TestShardedEmbeddingBagCollection(ShardedEmbeddingBagCollection):
```

Comment:
I think all attributes and methods of this class are the same as those of `ShardedEmbeddingBagCollection`. Why define a new one?

Reply:
You